README.md

```diff
@@ -47,86 +47,52 @@ First, create a copy of this repository under your own GitHub account.
-      ├── _index.md
-      └── <your-organization-uid>/
-          └── <your-learning-path>/
-              ├── _index.md
-              └── <your-course-1>/
-              └── <your-course-2>/
-                  ├── _index.md
-                  └── content/
-                      └── your-lesson-1.md
-                      └── your-lesson-2.md
-  ```
-
-  - **Delete the example content** inside `content/learning-paths/<your-organization-uid>/`.
-  - **Create your folder structure** following the example's hierarchy.
-  - **Add your lessons** as Markdown (`.md`) files inside the `content` directory of a course.
-  - **Use frontmatter** at the top of your `_index.md` and lesson files to define titles, descriptions, and weights.
+> **📚 Content Creation Guide:** For detailed instructions on structuring learning paths, writing courses, and creating assessments, see the [Content Creation Documentation](https://docs.layer5.io/cloud/academy/creating-content/).
 
-### 5. Add Assets (Images & Videos)
-
-Enhance your course with images and other visual aids. To ensure compatibility with the multi-tenant Academy platform, **do not use standard Markdown image links**. Instead, use the `usestatic` shortcode, which generates the correct, tenant-aware path for your assets.
-
-**How to Add an Image**
+The basic structure is: **Learning Path → Course → Chapter → Lesson**.
 
-1.  Place your image file (e.g., `hugo-logo.png`) in your scoped static directory:
-
-    ```text
-    static/<your-organization-uid>/images/hugo-logo.png
-    ```
-2.  In your `lesson-1.md` file, embed the image using the `usestatic` shortcode. The `path` is relative to your scoped static folder: 
+Quick setup steps:
+- Delete the example content inside `content/learning-paths/<your-organization-uuid>/`
+- Create your folder structure following the example's hierarchy
+- Add your lessons as Markdown files with proper frontmatter
 
-    ```text
-    ![The Hugo Logo]({{</* usestatic path="images/hugo-logo.png" */>}})
-    ```
-
-Then the system will automatically convert this into the correct URL when building the site.
+### 5. Add Assets (Images & Videos)
 
-**How to Add a Video**
+For this template, place assets in your scoped static directory: `static/<your-organization-uuid>/`
 
+Use the `usestatic` shortcode for images:
 ```text
-{{</* card 
-title="Video: Example" */>}}
-<video width="100%" height="100%" controls>
-    <source src="https://exmaple.mp4" type="video/mp4">
-    Your browser does not support the video tag.
-</video>
-{{</* /card */>}}
+![Image description]({{</* usestatic path="images/your-image.png" */>}})
 ```
 
+> **🎨 Asset Management:** For comprehensive guidelines on adding multimedia content, interactive elements, and assessments, see the [Content Creation Documentation](https://docs.layer5.io/cloud/academy/creating-content/).
+
 ### 6. Local Development
 
 To preview your content locally, run the Hugo server from the project root:
 
 ```bash
-hugo server
+make site

```

Comment:
`make setup` not needed first?

Why are we repeating the build steps here in this README?